spec/subnets-lifecycle.md

```diff
@@ -0,0 +1,68 @@
+# Subnet Lifecycle
+A subnet is a new subsystem that a user can spawn from a parent subnet in a permissionless and on-demand way, depending on scalability requirements. Subnets have separate consensus algorithms and cryptoeconomic rules from their parent subnet. Subnets are firewalled from the parent network.
+
+Subnets begin with a chosen "rootnet". In case of L2 subnets, “rootnets” refer to a layer 1 blockchain, such as Filecoin or Ethereum. Child subnets are spawned from the rootnet and the rootnet becomes the parent subnet.
```

Comment:
> Subnets begin with a chosen "rootnet". 

Parent subnet, parent network, rootnet – are these all the same concepts? Maybe it makes sense to have a Glossary? eg take a look at https://fluence.dev/docs/build/glossary.

Reply:
Good point, I can raise this to the team. But here `rootnet` simply means the `parent` of all, like the starting subnet, which for the sake of recurrence.